web_accounts_list.json

```diff
@@ -4591,6 +4591,61 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "Tunisie-Foot",

```

Comment:
I do not think WMN is right for this site. The reasons:
1. The site adds randominess/UID to usernames. So someone searches for `johndoe` and it won't find them because in the platform their username is something like `12435-johndoe`. So unless the WMN user knows that exact UID+username, this site will never find anything
2. We sometimes can do a search for a username and then use those results in WMN. But here, again, this platform is not good fit. because anything you search for will yield results.
3. My suggestion is to remove tunisie-foot.